packages/eas-cli/src/commands/update/roll-back-to-embedded.ts

```diff
@@ -0,0 +1,353 @@
+import { Platform as PublishPlatform } from '@expo/config';
+import { Errors, Flags } from '@oclif/core';
+import chalk from 'chalk';
+import nullthrows from 'nullthrows';
+
+import { ensureBranchExistsAsync } from '../../branch/queries';
+import { getUpdateGroupUrl } from '../../build/utils/url';
+import { ensureChannelExistsAsync } from '../../channel/queries';
+import EasCommand from '../../commandUtils/EasCommand';
+import { EasNonInteractiveAndJsonFlags } from '../../commandUtils/flags';
+import { getPaginatedQueryOptions } from '../../commandUtils/pagination';
+import fetch from '../../fetch';
+import {
+  PublishUpdateGroupInput,
+  StatuspageServiceName,
+  UpdatePublishMutation,
+} from '../../graphql/generated';
+import { PublishMutation } from '../../graphql/mutations/PublishMutation';
+import Log, { link } from '../../log';
+import { ora } from '../../ora';
+import { RequestedPlatform } from '../../platform';
+import { getOwnerAccountForProjectIdAsync } from '../../project/projectUtils';
+import {
+  ExpoCLIExportPlatformFlag,
+  defaultPublishPlatforms,
+  getBranchNameForCommandAsync,
+  getRequestedPlatform,
+  getRuntimeToPlatformMappingFromRuntimeVersions,
+  getRuntimeVersionObjectAsync,
+  getUpdateMessageForCommandAsync,
+} from '../../project/publish';
+import { ensureEASUpdateIsConfiguredAsync } from '../../update/configure';
+import { getUpdateGroupJsonInfo } from '../../update/utils';
+import {
+  checkDirectiveBodyAgainstUpdateInfoGroup,
+  getCodeSigningInfoAsync,
+  getDirectiveBodyAsync,
+  signBody,
+} from '../../utils/code-signing';
+import uniqBy from '../../utils/expodash/uniqBy';
+import formatFields from '../../utils/formatFields';
+import { enableJsonOutput, printJsonOnlyOutput } from '../../utils/json';
+import { maybeWarnAboutEasOutagesAsync } from '../../utils/statuspageService';
+import { getVcsClient } from '../../vcs';
+
+type RawUpdateFlags = {
+  auto: boolean;
+  branch?: string;
+  channel?: string;
+  message?: string;
+  platform: string;
+  'private-key-path'?: string;
+  'non-interactive': boolean;
+  json: boolean;
+};
+
+type UpdateFlags = {
+  auto: boolean;
+  platform: ExpoCLIExportPlatformFlag;
+  branchName?: string;
+  channelName?: string;
+  updateMessage?: string;
+  privateKeyPath?: string;
+  json: boolean;
+  nonInteractive: boolean;
+};
+
+export default class UpdateRollBackToEmbedded extends EasCommand {
+  static override description = 'roll back to the embedded update';
```

Comment:
Not really related to the PR, but can you please update the wording in https://github.com/expo/eas-cli/blob/main/packages/eas-cli/src/commands/update/republish.ts#L59? (rollback -> roll back)